### PR TITLE
chore(package): use @bitgo/utxo-lib

### PR DESCRIPTION
**Pull request by OttoAllmendinger**
Use new import path

Issue: BG-34381

### CODE REVIEW

File: app/sign.js [hunk 1]
```diff
@@ -131,12 +132,10 @@ const promptForConfirmationAndKey = function(recoveryRequest, outputs, skipConfi
  * Gets the backup private key that can be used to sign the transaction.
  * @param xprv The provided extended private key (BIP32).
  * @param expectedXpub The public key specified with the request.
- * @returns The private key to sign the transaction.
+ * @returns Buffer - the private key buffer to sign the transaction.
```
Comment (sangaman) [inline app/sign.js:135]: Should this be `{Buffer}` to denote a type?

File: app/sign.js [hunk 2]
```diff
@@ -421,20 +421,18 @@ const parseKey = co(function *(rawkey, coin, path) {
...
     const xprv = node.toBase58();
     return xprv;
 
   }
-  // if it doesn't have commas, we expect it is an xprv or xlmsecret properly formatted
   if (path) {
-    let node = utxoLib.HDNode.fromPrivateKeyBuffer(Buffer.from(rawkey, 'hex'));
-    node = node.derivePath(path);
-    return node.toBase58();
+    throw new Error(`cannot derive key with path`)
```
Comment (sangaman) [inline app/sign.js:433]: The related commit has a very nice comment: "There is no plausible use case where we want to instantiate a bip32 node from just the private key buffer (without chaincode)." I think something to that that effect (or even just the same exact wording) in a comment here would be helpful so we know why we can't derive with a path here.

**Review by sangaman — APPROVED** [review]
Looks good, just a few minor notes. This would replace #89 - and also is related to Ticket BG-33735 for the prova dependency removal.

I don't know why the build fails (same issue with the related PR above), however.

File: app/sign.js (hunk 1 above)
Comment (OttoAllmendinger) [reply app/sign.js:135]: intellij at least doesn't seem to need them

**Review by sangaman — APPROVED** [review]

**Pull request merged**